server/transaction_event_logger.go

```diff
@@ -149,6 +153,50 @@ func (tl *transactionEventLogger) Delete(ctx context.Context, s *schema.Schema,
 	return tl.logEvent(ctx, "delete", resource, configVersion)
 }
 
+func (tl *transactionEventLogger) DeleteFilter(ctx context.Context, s *schema.Schema, filter transaction.Filter) error {
+	if !tl.syncEventNeeded(s) {
```

Comment:
This is just a fast path, without it result would be the same (although slower). Am I right?

Reply:
Indeed